src/content/docs/installation/boot_managers.md

```diff
@@ -18,6 +18,25 @@ handles the task of loading an OS kernel into memory, often along with support f
 For a more detailed explanation and the reason both terms are often used interchangeably,
 please see "[Managing EFI Boot Loaders for Linux: Basic Principles](https://www.rodsbooks.com/efi-bootloaders/principles.html)" by Rod Smith.
 
+Here is the feature comparison table for the boot managers, accompanied by additional details and recommendations:
+
+### **Feature Comparison Table**
+
+| **Feature**                | **systemd-boot**                     | **rEFInd**                            | **GRUB**                           |
+|----------------------------|---------------------------------------|---------------------------------------|------------------------------------|
+| **UEFI Support**            | ✅ Yes                               | ✅ Yes                               | ✅ Yes                            |
+| **BIOS Support**            | ❌ No                                | ❌ No                                | ✅ Yes                            |
+| **Autodetection**           | ❌ Limited (manual configuration)    | ✅ Full autodetection                | ✅ Autodetection (regenerates config) |
+| **Theming/Customization**   | ❌ Minimal                           | ✅ Extensive                          | ✅ Extensive                      |
+| **Configuration Complexity**| Simple                              | Moderate                             | Complex                          |
+| **Secure Boot Support**     | ✅ Yes                               | ✅ Yes                               | ✅ Yes                            |
+| **Filesystem Support**      | FAT, FAT16, FAT32                   | FAT, EXT4, BTRFS                     | Most Linux FS                     |
```

Comment:
While it is true that GRUB has support for loads of filesystems, I believe that the support is strictly read-only. For example, see https://github.com/CachyOS/distribution/issues/59 where GRUB fails to write `grub.env` to a BTRFS filesystem.

Reply:
I’m not sure how to solve this at the moment, but I can try investigating which filesystems are specifically supported by GRUB.

Reply:
It is likely sufficient to add a footnote clarifying that certain filesystems aren't supported (right not those that come to mind are BTRFS and ZFS).

Reply:
https://www.gnu.org/software/grub/manual/grub/html_node/Features.html?

Support multiple filesystem types transparently, plus a useful explicit blocklist notation. The currently supported filesystem types are Amiga Fast FileSystem (AFFS), AtheOS fs, BeFS, BtrFS (including raid0, raid1, raid10, gzip and lzo), cpio (little- and big-endian bin, odc and newc variants), Linux ext2/ext3/ext4, DOS FAT12/FAT16/FAT32, exFAT, F2FS, HFS, HFS+, ISO9660 (including Joliet, Rock-ridge and multi-chunk files), JFS, Minix fs (versions 1, 2 and 3), nilfs2, NTFS (including compression), ReiserFS, ROMFS, Amiga Smart FileSystem (SFS), Squash4, tar, UDF, BSD UFS/UFS2, XFS, and ZFS (including lzjb, gzip, zle, mirror, stripe, raidz1/2/3 and encryption in AES-CCM and AES-GCM).

btw: BTRFS should works, idk what happened.

Reply:
As I mentioned previously, the support mentioned here seems to indicate a sufficient condition for reading, not writing. The issue seems to be that GRUB doesn't support sparse files, which BTRFS and some others do from compression or so.